datajunction-server/datajunction_server/internal/authentication/jwt.py

```diff
@@ -68,3 +79,13 @@ def decrypt(value: str) -> str:
     """
     settings = get_settings()
     return jwe.decrypt(value, settings.secret).decode("utf-8")
+
+
+async def get_token(
+    authorization: str = Header(default="Bearer "),
+) -> str:
+    """
+    Get an authorization bearer token from an authorization header
+    """
+    _, token = authorization.split(" ")
```

Comment:
I couldn't find anything, but I'd think there'd be a helper function in FastAPI that returns the token so we don't have to do this parsing ourselves?

Reply:
It should be possible to do something like this right?

```
from fastapi.security.http import HTTPAuthorizationCredentials, HTTPBearer

async def get_token(
    authorization: Optional[HTTPAuthorizationCredentials] = Depends(HTTPBearer(auto_error=False)),
) -> str:
    if auth is None:
        raise DJException(
            http_status_code=HTTPStatus.UNAUTHORIZED,
            errors=DJError(
                code=ErrorCode.OAUTH_ERROR,
                message=" ... ",
            ),
        )
    return auth.credentials
```

Reply:
Nice! I used `fastapi.security.utils.get_authorization_scheme_param` exactly how the `HTTPBearer` class does in the `DJHTTPBearer` wrapper, so it's hitting the same logic as this example. 👍